src/PrestaShopBundle/Routing/Converter/AbstractLegacyRouteProvider.php

```diff
@@ -126,4 +135,22 @@ private function initControllerActions()
             }
         }
     }
+
+    /**
+     * @param string $controller
+     *
+     * @return array|null
+     */
+    private function getControllerActions($controller)
+    {
+        $controllerActions = null;
+        foreach ($this->controllersActions as $listController => $actions) {
+            if (strtolower($listController) == strtolower($controller)) {
+                $controllerActions = $actions;
+                break;
+            }
+        }
+
+        return $controllerActions;
```

Comment:
In this method you can avoid setting a value to return. You directly return `$actions` if you find it, and at the end `return null;`.

Reply:
Sorry didn't have time to fix it ^^